pandas/tests/test_downstream.py

```diff
@@ -274,3 +274,23 @@ def __radd__(self, other):
 
     assert right.__add__(left) is NotImplemented
     assert right + left is left
+
+
+@td.skip_if_no("pydantic")
+@td.skip_if_no("pydantic_core")
+@pytest.mark.parametrize("series", [
+    Series([1, 2, 3]),
+    Series(np.array([1, 2, 3])),
+    Series({'a': 1, 'b': 2, 'c': 3}),
+    Series(3),
+])
+def test_pydantic_protocol(series: Series) -> None:
+    from pydantic import BaseModel
+
+    class Model(BaseModel):
+        series: Series
+
+
+    model = Model(series=series)
+    assert model.model_dump() == {}
+    assert model.model_json_schema() == {}
```

Comment:
could you explain these two lines please?

Reply:
ok I'd suggest following https://pandas.pydata.org/docs/dev/development/contributing_environment.html